<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<script src="site_libs/htmlwidgets-0.7/htmlwidgets.js"></script>
<script src="site_libs/datatables-binding-0.2/datatables.js"></script>
<link href="site_libs/dt-core-1.10.12/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.12/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.12/js/jquery.dataTables.min.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 45px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 50px;
  margin-top: -50px;
}

.section h2 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h3 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h4 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h5 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h6 {
  padding-top: 50px;
  margin-top: -50px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">DIEGO DIEZ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-book"></span>
     
    Publications
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    <span class="fa fa-cog"></span>
     
    Packages
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="q3tools/index.html">q3tools</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">
    <span class="fa fa-info"></span>
     
    About
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/ddiez">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<p><br><br></p>
<p>Here I show my publication record and the number of citations per year. Mostly I wanted to use the awesome <code>scholar</code> R package to programmatically extract and analyze <a href="http://scholar.google.com">Google Scholar</a> profiles. If you are interested on the <code>scholar</code> package you can get more information <a href="https://github.com/jkeirstead/scholar">here</a>.</p>
<div id="citations-per-year" class="section level2">
<h2>Citations per year</h2>
<p><img src="publications_files/figure-html/unnamed-chunk-3-1.png" width="384" style="display: block; margin: auto;" /></p>
</div>
<div id="list-of-publications" class="section level2">
<h2>List of publications</h2>
<div id="htmlwidget-86127a3cf3689c03c520" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-86127a3cf3689c03c520">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22"],[2016,2016,2015,2014,2014,2013,2013,2013,2013,2013,2012,2011,2011,2010,2010,2010,2010,2009,2009,2008,2008,2007],[0,0,12,15,13,47,34,25,10,8,13,34,20,70,68,42,5,86,9,57,19,20],["Exploring the interactions of the RAS family in the human protein network and their potential implications in RAS-directed therapies.","A novel affinity-based method for the isolation of highly purified extracellular vesicles","Neuronal exosomes facilitate synaptic pruning by up-regulating complement factors in microglia","Network analysis in the investigation of chronic respiratory diseases. From basics to application","Systematic identification of transcriptional regulatory modules from protein–protein interaction networks","The IL-10/STAT3-mediated anti-inflammatory response: recent developments and future challenges","Distinct transcriptional regulatory modules underlie STAT3’s cell type-independent and cell type-specific functions","The repertoires of ubiquitinating and deubiquitinating enzymes in eukaryotic genomes","Genomic and computational approaches to dissect the mechanisms of STAT3’s universal and cell type-specific functions","PWMEnrich: PWM enrichment analysis","Network analysis identifies a putative role for the PPAR and type 1 interferon pathways in glucocorticoid actions in asthmatics","Lack of action of exogenously administered T3 on the fetal rat brain despite expression of the monocarboxylate transporter 8","Evolutionary expansion of the Ras switch regulatory module in eukaryotes","The use of network analyses for elucidating mechanisms in cardiovascular disease","Thyroid hormone-regulated mouse cerebral cortex genes are differentially dependent on the source of the hormone: a study in monocarboxylate transporter-8-and deiodinase-2-deficient mice","Thyroid hormone regulation of gene expression in the developing rat fetal cerebral cortex: prominent role of the Ca2+/calmodulin-dependent protein kinase IV pathway","varDB: A database of antigenic variant sequences—Current status and future prospects","Systems biology approaches and pathway tools for investigating cardiovascular disease","GeneRegionScan: a Bioconductor package for probe-level analysis of specific, small regions of the genome","Thyroid hormone action in the adult brain: gene expression profiling of the effects of single and multiple doses of triiodo-L-thyronine in the rat striatum","varDB: a pathogen-specific sequence database of protein families involved in antigenic variation","Codelink: an R package for analysis of GE healthcare gene expression bioarrays"],["Oncotarget","Scientific Reports","Scientific reports","American journal of respiratory and critical care medicine","Nucleic acids research","Briefings in functional genomics","Nucleic Acids Research","Molecular biology and evolution","Jak-Stat","R package version","BMC medical genomics","Endocrinology","Nucleic acids research","Mol. BioSyst.","Endocrinology","Endocrinology","Acta tropica","Mol. BioSyst.","Bioinformatics","Endocrinology","Bioinformatics","Bioinformatics"],["A Bueno, I Morilla, D Diez, AA Moya-Garcia, J Lozano, JA Ranea","W Nakai, T Yoshida, D Diez, Y Miyatake, T Nishibu, N Imawaka, K Naruse, ...","I Bahrini, J Song, D Diez, R Hanayama","D Diez, A Agustí, CE Wheelock","D Diez, AP Hutchins, D Miranda-Saavedra","AP Hutchins, D Diez, D Miranda-Saavedra","AP Hutchins, D Diez, Y Takahashi, S Ahmad, R Jauch, ML Tremblay, ...","AP Hutchins, S Liu, D Diez, D Miranda-Saavedra","AP Hutchins, D Diez, D Miranda-Saavedra","R Stojnic, D Diez","D Diez, S Goto, JV Fahy, DJ Erle, PG Woodruff, ÅM Wheelock, ...","C Grijota-Martínez, D Díez, G Morreale de Escobar, J Bernal, B Morte","D Díez, F Sánchez-Jiménez, JAG Ranea","D Diez, ÅM Wheelock, S Goto, JZ Haeggström, G Paulsson-Berne, ...","B Morte, A Ceballos, D Diez, C Grijota-Martínez, AM Dumitrescu, ...","B Morte, D Díez, E Ausó, MM Belinchón, P Gil-Ibáñez, C Grijota-Martínez, ...","D Diez, N Hayes, N Joannin, J Normark, M Kanehisa, M Wahlgren, ...","CE Wheelock, ÅM Wheelock, S Kawashima, D Diez, M Kanehisa, ...","L Folkersen, D Diez, CE Wheelock, JZ Haeggström, S Goto, P Eriksson, ...","D Diez, C Grijota-Martinez, P Agretti, G De Marco, M Tonacchera, ...","CN Hayes, D Diez, N Joannin, W Honda, M Kanehisa, M Wahlgren, ...","D Diez, R Alvarez, A Dopazo"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> \u003c/th>\n      <th>year\u003c/th>\n      <th>cites\u003c/th>\n      <th>title\u003c/th>\n      <th>journal\u003c/th>\n      <th>author\u003c/th>\n    \u003c/tr>\n  \u003c/thead>\n\u003c/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
